### PR TITLE
Fix gradient background height

### LINKED ISSUE #38: Update gradient background styling
**Is your feature request related to a problem? Please describe.**
Gradient background doesn't cover the whole page throughout the app.

**Describe the solution you'd like**
Fix the background height by changing to 100% instead 100vh.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Added div with className to wrap around h2 element.
2. Gave div 100vh to have gradient background cover whole screen vertically.

## Purpose
Fix background styling.

## Approach
Added height property to div wrapper to have it cover whole page.

Closes #38

### CODE REVIEW

File: movie-app/src/App.js [hunk 1]
```diff
@@ -45,10 +45,12 @@ function App() {
...
             setModalOpen={setModalOpen}
           />
-        ) 
-        : 
+        )
+        :
         (
+         <div className="error-container">
           <h2 className="error-text"> Please enter a valid movie title </h2>
+          </div>
```
Comment (Lap343) [inline movie-app/src/App.js:53]: This is all we had to do then nice

**Review by Lap343 — APPROVED** [review]
Quick code but works. Nice

**Review by gonzalespaulb — APPROVED** [review]
LGTM

**Review by Gjhnsn — APPROVED** [review]
Looks good!

**Pull request merged**